### ISSUE TITLE
Group invite links

### ISSUE THREAD

**Issue by mjneff2** [oss-apps/split-pro#54]
Clicking groups invite link is confusing when you're not already logged in. It should prob you to sign in rather than just sending you back to route `/`. Also seems to work a bit inconsistently even when logged in.

Side UX note on this, should probably redirect to the group you joined after successful joining, rather than to `/groups` page

**Comment by KMKoushik** [comment]
@mjneff2 I agree, group invite defenitely need some improvements. When not logged in, it should redirect to the group page once you login. I'll try to improve it thank you for raising this. You're helping in making the product better!
